amd_openvx_extensions/amd_rpp/include/internal_rpp.h

```diff
@@ -92,7 +92,8 @@ vx_node createNode(vx_graph graph, vx_enum kernelEnum, vx_reference params[], vx
 vx_status createRPPHandle(vx_node node, vxRppHandle ** pHandle, Rpp32u batchSize, Rpp32u deviceType);
 vx_status releaseRPPHandle(vx_node node, vxRppHandle * handle, Rpp32u deviceType);
 void fillDescriptionPtrfromDims(RpptDescPtr &descPtr, vxTensorLayout layout, size_t *tensorDims);
-void fillAudioDescriptionPtrFromDims(RpptDescPtr &descPtr, size_t *tensorDims, vxTensorLayout layout = vxTensorLayout::VX_NHW);
+void fillGenericDescriptionPtrfromDims(RpptGenericDescPtr &genericDescPtr, vxTensorLayout layout, size_t *maxTensorDims);
```

Comment:
why do we need line 94 & 95? Isn't line 94 for Generic Nodes and 96 for Audio

Reply:
Not sure why there are 3 different functions for fillDescriptions? We need to to converge all into one

Reply:
Will issue another PR for this . Since it requires changes in all the Audio Files